util/util_nix.go

```diff
@@ -74,18 +73,7 @@ func GetCurrentProcessTimes() (utime int64, stime int64, err error) {
 	return
 }
 
-// GetTotalMemory gets total system memory on Linux systems
+// GetTotalMemory gets total system memory
```

Comment:
I may be misunderstanding Go's build rules, but I think this will get compiled with the windows implementation, and you'll have two `GetTotalMemory` symbols.  I think this file can be deleted, and the implementations in `darwin` and `linux` can be capitalized.

Reply:
well I rolled back the _nix rename, confusing, ok

Reply:
We can just get rid of it, can't we? The `_linux` and `_darwin` can implement `GetTotalMemory`, just like the `_windows` file. Then there are no build tags at all.

Reply:
we can. I wanted to keep all exported methods in util.go (still need this file because other existing shared implementations) but it is probably does not matter much.

Reply:
`_windows` has the exported version though, that's what confused me. I think that as it stands, we would be defining the exported name in both, when compiling on windows.

Reply:
I guess I'm wrong because of build tags in `util.go`. 

I see what you were doing with `_nix` now. Consolidating the shared stuff.

Reply:
no we don't:
1. `util.go` has the following build tags:
    ```
    //go:build !windows
    // +build !windows
    ```
2. Windows build [passed](https://github.com/algorand/go-algorand/actions/runs/13794854062/job/38583858879?pr=6272#step:7:17) just fine